src/layouts/default.vue

```diff
@@ -207,9 +207,12 @@ export default {
 </script>
 
 <style scoped>
+.app {
+  @apply min-h-screen min-h-[100dvh];
+}
 .sidebar {
   /* Header height above md is 80px plus 1px for bottom border */
-  height: calc(100vh - 81px);
+  @apply h-[calc(100vh-81px)] h-[calc(100dvh-81px)];
```

Comment:
```suggestion
  @apply h-[calc(100vh - 81px)] h-[calc(100dvh - 81px)];
```

These calc statements need spaces between the operators to work.

Reply:
Tailwind says they need underscores: https://tailwindcss.com/docs/adding-custom-styles#handling-whitespace
But the arbitrary values can also use no whitespace at all: https://stackoverflow.com/questions/65976223/how-to-use-calc-in-tailwind-css. I think the "no whitespace" version looks cleaner than the "undescores" one.

Reply:
Oh I see, tailwind will produce the correctly-spaced result in the css. I agree this looks cleaner.